package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "smarthouse-frontend",
+  "name": "smart-house-frontend",
```

Comment:
If you start change this file, it will be better to move all dependencies to devDependencies zone to increase install speed of this package